src/imitation/scripts/ingredients/sqil.py

```diff
@@ -0,0 +1,48 @@
+"""This ingredient provides a SQIL algorithm instance."""
+import sacred
+from stable_baselines3 import dqn as dqn_algorithm
+
+from imitation.policies import base
+from imitation.scripts.ingredients import policy, rl
+
+sqil_ingredient = sacred.Ingredient(
+    "sqil",
+    ingredients=[rl.rl_ingredient, policy.policy_ingredient],
+)
+
+
+@sqil_ingredient.config
+def config():
+    total_timesteps = 3e5
+    train_kwargs = dict(
+        log_interval=4,  # Number of updates between Tensorboard/stdout logs
+        progress_bar=True,
+    )
+
+    locals()  # quieten flake8 unused variable warning
+
+
+@rl.rl_ingredient.config_hook
```

Comment:
I tried doing this originally, but it didn't work. For some reason it would set the variables inside of sqil. E.g. it would set `config["sqil"]["rl"]["rl_cls"]` instead of `config["rl"]["rl_cls"]` as intended.

Reply:
Can confirm having it in SQIL ingredient will set variables inside of SQIL. More problematically moving hook to train_imitation experiment seems to then have no effect on sub-ingredients. So, ugly though it is, I think we probably do need to keep it here. Good news is these hooks are no-op when command name is not sqil.